Public/Src/Utilities/Configuration/Mutable/SandboxConfiguration.cs

```diff
@@ -16,7 +16,7 @@ public SandboxConfiguration()
             m_unsafeSandboxConfig = new UnsafeSandboxConfiguration();
 
             FailUnexpectedFileAccesses = true;
-            DefaultTimeout = 10 * 60 * 1000;
+            DefaultTimeout = 15 * 60 * 1000;

```

Comment:
This is going to change the default timeout for all pips in all builds, right?  If so, I think this default value is also mentioned in the help text, so it should be updated there as well

Reply:
Help text is just a generic message explaining the value semantics.